core/src/test/scala/unit/kafka/log/LogTest.scala

```diff
@@ -2137,6 +2137,7 @@ class LogTest {
     for (_ <- 0 until 100)
       log.appendAsLeader(createRecords, leaderEpoch = 0)
 
+    log.close()
```

Comment:
Do we also want to have a test where we `delete` without calling `close`?

Reply:
It seems that we already have a couple of tests in LogTest where `log.delete()` is called without calling `close`. So I guess it is fine. But I can revert the change here and add a new to call delete() after close(). What do you think?

Reply:
The only thing is that the other tests don't assert anything after the `delete` call. We could add asserts to the other tests or add a new test to make sure we're not losing coverage for that case.

Reply:
Sure. I have updated the patch to add a new test.